src/helpers/format.ts

```diff
@@ -758,3 +759,15 @@ function convertInternalFormatToFormat(internalFormat: InternalFormat): Format {
   }
   return format;
 }
+
+/** Transform the value into a date in the locale format */
+export function formatDate(value: string, locale: Locale): string {
+  if (isNumber(value, locale)) {
+    return formatValue(toNumber(value, locale), { format: locale.dateFormat, locale });
+  }
+  const internalDate = parseDateTime(value, locale);
+  if (!internalDate) {
+    return "";
+  }
+  return formatValue(internalDate.value, { format: locale.dateFormat, locale });
```

Comment:
can be simplified to this, but there's a f***ing circular dependency when importing `parseLiteral` :grimacing: 
Maybe it could just be in-lined since its used only once and it's quite specific (returning empty string if it's not a date is most probably not a common generic behavior)
```ts
  const value = parseLiteral(valueStr, locale);
  if (typeof value === "number") {
    return formatValue(value, { format: locale.dateFormat, locale });
  }
  return ""
```